src/app/components/flyout/flyout-guidelines.component.html

```diff
@@ -15,7 +15,7 @@
 	
 	<stache-row>
     <stache-column screenSmall="6">
-			<p>Use a flyout to open records in a list so that users can return to the list view after completing quick tasks on the record pages.</p>
+			<p>Use a flyout to open records in a list so that users can return to the list view after completing quick tasks on the record pages. If the user will need a full-screen view of details or will be taking multiple actions on the record, navigate to a full page instead.</p>
```

Comment:
The new sentence also appears in the When not to use section. I'm not sure if we should keep it to really drive the point home, or if it just comes across as redundant.

Reply:
It started in the Do Not section but got overlooked so we wanted to double up to ensure it gets across

Reply:
I'd tweak the opening clause to make "user" plural and use present tense verbs: "If users require a full-screen view of details or need to take multiple actions on the record,"